extern/sector-storage/stores/local.go

```diff
@@ -560,6 +570,8 @@ func (st *Local) Remove(ctx context.Context, sid abi.SectorID, typ storiface.Sec
 	return nil
 }
 
+// RemoveCopies removes all copies of the given sector file types for the given sector from the local sealing scratch space.
+// However, it does NOT remove the "primary" copies of those sector files i.e. sector files that are stored in the local long term storage space.
```

Comment:
Primary copies can also live in sealing storage while the sector is going through the sealing pipeline. "Primary" basically just marks the copy which shouldn't be removed by this method.